dracut/51coreos-installer/module-setup.sh

```diff
@@ -32,30 +35,23 @@ install() {
     xfs_growfs \
     wc         \
     lsblk      \
-    wipefs
-
-    inst_multiple \
-        awk       \
-        cat       \
-        dd        \
-        grep      \
-        mktemp    \
-        partx     \
-        rm        \
-        sed       \
-        sfdisk    \
-        sgdisk    \
-        find
-        
-    inst_multiple -o \
-        clevis-encrypt-sss \
-        clevis-encrypt-tang \
-        clevis-encrypt-tpm2 \
-        clevis-luks-bind \
-        clevis-luks-common-functions \
-        clevis-luks-unlock \
-        pwmake \
-        tpm2_create
+    wipefs     \
+    awk        \
+    cat        \
+    dd         \
+    grep       \
+    mktemp     \
+    partx      \
+    rm         \
+    sed        \
+    sfdisk     \
+    sgdisk     \
+    find       \
+    gpg-connect-agent \
+    clevis-luks-unlock
+    # not needed if clevis-dracut ships with 60clevis-pin-null
+    # just depends on clevis-pin-null above at the beginning
+    #clevis-decrypt-null \
```

Comment:
doesn't need depend, installing `clevis-dracut` and requiring clevis should be enough to pull all pins in